modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -648,6 +648,8 @@ definition {
 	@description = "LPS-149508 - Verify if there are default resources of the System Objects on the Audit Portlet"
 	@priority = "4"
 	test CanViewDefaultResourcesOnSystemObjects {
+		property portal.upstream = "quarantine";
+
 		Audit.openAuditAdmin();
 
```

Comment:
@andreefarias please quarantine tests in a separate commit in the future. We really shouldn't be combining test fixes with test quarantines because it makes it really hard to track.